src/Main/TheoryLoader.hs

```diff
@@ -317,13 +319,14 @@ loadTheory thyOpts input inFile = do
 
     parse p = parseString (toParserFlags thyOpts) inFile p input
 
-    translate | isParseOnlyMode = return
-              | otherwise       = Sapic.typeTheory
+    translate | isMSRModule   = Sapic.typeTheory
```

Comment:
@charlie-j I mainly turned the negation around here. The rest is commenting and code maintenance.